lib/maxrectpacker.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
I will try to package it in npm module after finished the fine-tuning of these codes. This is my first node.js project, still a lot to learn ;)